aws/table_aws_elastic_beanstalk_environment.go

```diff
@@ -0,0 +1,301 @@
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "application_name",
+				Description: "The name of the application associated with this environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "cname",
+				Description: "The URL to the CNAME for this environment.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsElasticBeanstalkEnvironment,
+			},
+			{
+				Name:        "date_updated",
+				Description: "The last modified date for this environment.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "endpoint_url",
+				Description: "The URL to the LoadBalancer.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsElasticBeanstalkEnvironment,
+			},
+			{
+				Name:        "health",
+				Description: "The health status of the environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "health_status",
+				Description: "Returns the health status of the application running in your environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "operations_role",
+				Description: "The Amazon Resource Name (ARN) of the environment's operations role.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "platform_arn",
+				Description: "The ARN of the platform version.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "solution_stack_name",
+				Description: "The name of the SolutionStack deployed with this environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "status",
+				Description: "The current operational status of the environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "template_name",
+				Description: "The name of the configuration template used to originally launch this environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "version_label",
+				Description: "The application version deployed in this environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "environment_links",
+				Description: "A list of links to other environments in the same group.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsElasticBeanstalkEnvironment,
+			},
+			{
+				Name:        "resources",
+				Description: "The description of the AWS resources used by this environment.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags assigned to the Repository",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     listElasticBeanstalkEnvironmentTags,
+				Transform:   transform.FromField("ResourceTags"),
+			},
+			{
+				Name:        "tier",
+				Description: "Describes the current tier of this environment.",
+				Type:        proto.ColumnType_JSON,
+			},
+
```

Comment:
{
				Name:        "tier",
				Description: "Describes the current tier of this environment.",
				Type:        proto.ColumnType_JSON,
			},
			{
				Name:        "tags_src",
				Description: "A list of tags assigned to the Repository",
				Type:        proto.ColumnType_JSON,
				Hydrate:     listElasticBeanstalkEnvironmentTags,
				Transform:   transform.FromField("ResourceTags"),
			},

Reply:
we usually keep the tags_src as the last column(but before the standard columns)